osi_common.proto

```diff
@@ -699,13 +699,13 @@ message ColorDescription
 }
 
 //
-// \brief Greyscale color model
+// \brief Grayscale color model
 //
-// ColorGrey defines a greyscale.
+// ColorGrey defines a grayscale.
 //
 message ColorGrey
```

Comment:
@pmai do we need an Alias here as well?

Reply:
Wait, let me guess: Alias only works for ENUMS and not signals? Should we add a TODO here? Does it bother?